config/environments/development.rb

```diff
@@ -1,5 +1,7 @@
 require "#{Rails.root}/lib/util/exception_notifier.rb"
 
+I18n.config.available_locales = :en
```

Comment:
I'm not sure why we need this, but something was erroring because it was missing. We have it in CAS already: https://github.com/greenriver/boston-cas/blob/fcb9bbc003eb67c1314e303189ccddbcb2b4b49c/config/environments/development.rb#L124